plugins/reach/lib/model/data/VendorVideoAnalysisCatalogItem.php

```diff
@@ -20,4 +23,33 @@ public function isEntryTypeSupported($type, $mediaType = null): bool
 		$supportedMediaTypes = [entry::ENTRY_MEDIA_TYPE_VIDEO];
 		return $type === entryType::MEDIA_CLIP && in_array($mediaType, $supportedMediaTypes);
 	}
+
+	public function setVideoAnalysisType($v)
+	{
+		$this->putInCustomData(self::CUSTOM_DATA_VIDEO_ANALYSIS_TYPE, $v);
+	}
+
+	public function getVideoAnalysisType()
+	{
+		return $this->getFromCustomData(self::CUSTOM_DATA_VIDEO_ANALYSIS_TYPE, null, true);
```

Comment:
why putting true here as the default values?
i assume you want it to be parsed to VendorVideoAnalysisType::OCR so better just mention it with the enum